src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -421,7 +428,7 @@ protected function build(ContainerBuilder $container)
      */
     protected function getContainerClass()
     {
-        return $this->name.ucfirst($this->environment).($this->debug ? 'Debug' : '').'ProjectContainer';
+        return str_replace('\\', '_', \get_class($this)).ucfirst($this->environment).($this->debug ? 'Debug' : '').'Container';
```

Comment:
This change breaks BC by removing the "name" from the dimensions that compose separate namespaces.
This is why the CI is red (and was also caught before merging ;) )
Side note: the container class is also a legacy concept; its real name is random (it's the hash of its contents actually.)
The only real remaining use case of the container class is for generating the *name* of the file that serves as entry point for loading the container.